functions/brinks/src/lib/stripe/validateCheckoutSession.js

```diff
@@ -0,0 +1,68 @@
+import debug from 'debug';
+import * as Sentry from '@sentry/node';
+import { dataSources } from '@thatconference/api';
+import { Firestore } from '@google-cloud/firestore';
+
+const dlog = debug('that:api:functions:brinks:validateCheckoutSession');
+const memberStore = dataSources.cloudFirestore.member;
+const productStore = dataSources.cloudFirestore.product;
+const firestore = new Firestore();
```

Comment:
I'm not sure if we're recreating this anywhere else across brinks. I'mo wondering if we should just create this instance once using express.set([instance]) and then pull it off with req.app.get('[thing])? I'm guessing it's expensive to create that connection.

Reply:
Not a connection at this point, only a reference to the function. We could consolidate any such references to a request reference if you feel it will be better. It is something I have not done so don't have a good opinion on. Being a function reference I don't see it being 'expensive'.

Reply:
I did some reading on expressjs and the `app.set()` and `app.get()` functions. I didn't know these existed. Lets discuss what I am thinking and see if makes sense and aligns with your suggested approach.

Reply:
In `src/index.js` simple enough to instantiate the firestore object and store it using `app.set('firestore', firestore)`. The part I am not getting is how do I access that value from middleware? It seems that it is only accessible from the app (express) object.

Reply:
refactored in brinks: ac099c2